playwright/pages/giftCardsPage.ts

```diff
@@ -45,18 +54,22 @@ export class GiftCardsPage extends BasePage {
   async clickDeactivateButton() {
     await this.deactivateButton.click();
   }
+  async clickExportGiftCards() {
+    await this.showMoreMenuButton.click();
```

Comment:
Not sure is it best solution. I think `    await this.showMoreMenuButton.click();
` 
could be separate action in test body for the sake of clarity when reading test definitions. But I leave it up to you